Mathlib/Analysis/NormedSpace/Star/ContinuousFunctionalCalculus/Instances.lean

```diff
@@ -282,6 +283,25 @@ lemma spectrum_star_mul_self_nonneg {b : A} : ∀ x ∈ spectrum ℝ (star b * b
   have h_eqOn := eqOn_of_cfc_eq_cfc (a := star b * b) h_eq_a_neg
   simpa [sup_eq_left.mpr hx'.le] using h_eqOn hx
 
+lemma IsSelfAdjoint.le_algebraMap_norm_self [PartialOrder A] [StarOrderedRing A] (a : A)
+    (ha : IsSelfAdjoint a := by cfc_tac) : a ≤ algebraMap ℝ A ‖a‖ := by
```

Comment:
Also, can we get the non-unital version? That is, `{a : A} (ha : IsSelfAdjoint a) (b : A) : star b * a * b ≤ ‖a‖ • star b * b`? You can prove it by passing to the unitization. (It's possible you need mine an Anatole's work first, in which case we can leave it to a future PR.)

Reply:
I'll try it and see how it goes.

Reply:
I'll take your word for it and leave it for another PR.